storage/remote/queue_manager.go

```diff
@@ -409,6 +410,7 @@ type QueueManager struct {
 	relabelConfigs       []*relabel.Config
 	sendExemplars        bool
 	sendNativeHistograms bool
+	sendMetadata         bool
```

Comment:
```suggestion
	sendWALMetadata         bool
```

Reply:
will stop marking those

Reply:
again, what do you think of just removing the option here, and always sending WAL metadata for the 2.0 format?